src/style.css

```diff
@@ -62,6 +82,23 @@ h3 {
     'Open Sans', 'Helvetica Neue', sans-serif;
   font-size: 45px;
 }
+.Series {
+  height: content;
+  display: flex;
+  justify-content: center;
+  align-content: start;
+  flex-wrap: wrap;
+}
+.Series-box {

```

Comment:
I would like to have the CSS class schema be kebab-case see what the different cases are [here](https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841#749b)